pug/pages/product-list.pug

```diff
@@ -0,0 +1,45 @@
+    - const pages = ["Trang trước", "1", "2", "3", "Trang cuối"]
```

Comment:
Bỏ các biến sang 1 trang khác nhé. Rồi include vào sử dụng chung listData ở cả 2 trang product grid và product list luôn
Ví dụ: productVariables.pug

Reply:
2 list này khác nhau anh ạ
product grid không có phần description ạ

Reply:
k có thì k in ra description, đâu phải cứ đặt biến là phải dùng hết đâu e